Log/v1.5_Changes.md

```diff
@@ -0,0 +1,51 @@
+# Version 1.5 Changes
+
+For many many reasons, we're making some significant changes to the program that will go into effect starting June 19, 2017.
+
+This document outlines the planned changes from a high level. More details to follow.
+
+## What's going away
+
+- Stats (i.e. most of what the "game" formerly was)
+- Levels
+- Wandering in the wilderness (lack of learning tracks)
+
+## What's staying
+
+- Project-based learning (at later phases)
+- Apprenticeship (at later phases)
+- Focus on skill-building
+- Free coffee
+
+## New stuff
+
+- Phases (5 of them, designed to be completed in sequence)
+- More support (appropriate to the phase)
+- Phase assessments
+- New membership review process based on assessments
+- Tighter scaffolding & structure at earlier phases
+- More career prep & support in final phase
+
+## Where to find more information
+
+To learn about Phases and Assessments, check out the new [Phases section](/Phases).
+
+More information will be shared as it becomes available. If you have questions in the meantime, ask a Moderator.
+
+## The Transition
+
+Moving from a loosely sequenced program to a more structured one is not a simple process. How can we know which learners should be in which phase?
+
+To solve this problem, during the week of 6/19 _everyone_ will take one of the three [Phase Assessments](/Phases/Assessments). Based on the outcome of this assessment, learners will be placed into either Phase 1, 2, 3, 4, or 5.
```

Comment:
We won't _place_ anyone into Phase 5, but people beyond 32 weeks can choose Phase 5 over Phase 4 if they want to.